packages/react/src/SubdomainNavBar/SubdomainNavBar.tsx

```diff
@@ -369,6 +450,15 @@ const _SearchInternal = (
                 </ul>
               </div>
             )}
+            <div
+              aria-live="polite"
+              aria-atomic="true"
```

Comment:
Just for my own understanding, is `true` preferred to `false` for any particular reason? If search results were particularly large for example, and a user is trying to optimise / refine a search query, would it not be better to read out the changes rather than the entire result again? The top result for example, may not be any different.

Reply:
Good question! The intention was so that the user would be notified of changes even if their query amounted in the same number of results. `aria-atomic` will ensure that it announces on each change regardless of if the value inside the live region is the same as any prior announcements. (e.g. user types "devops" AT announces "**8 suggestions**", user types "devo", amount of results haven't changed so AT announces "**8 suggestions**" again), without the `aria-atomic="true"`, the announcements would most likely cut out after the initial "8 suggestions" announcement.


There definitely is some redundancy here, but the `aria-live="polite"` should make it not interrupt anything else the user might be doing.